bin/mkrc.in

```diff
@@ -84,6 +84,9 @@ fi
 
 files=$@
 
+PREVIFS=$IFS
+IFS=$(echo -en "\n\b")
```

Comment:
`echo`, especially with options, especially with these options, is a "portability train wreck":

- http://wiki.bash-hackers.org/commands/builtin/echo
- http://cfajohnson.com/shell/cus-faq.html#Q0b
- http://www.in-ulm.de/~mascheck/various/echo+printf/

`printf "\n\b"` would probably work, but I wonder if the simpler

````sh
IFS="
"
```

(as mike linked to in his comment) would be best.